### DISCUSSION TITLE
Just a few questions as a newbie

### DISCUSSION THREAD

**Discussion by shahiljain** [focus-editor/focus#137] [Q&A]
Hey, randomly came across this project on youtube, looks fanstastic. Perfect replacement for something like the default notepad (absolutely shit compared to the default stuff on linux). Anyways I just had a few questions and they aren't really issues so here I am.

### 1. How to change default window size?
Pretty self explanatory I think.

### 2. Is there an option to add language?
I work with auto hotkey alot and I would love to know if I can get that syntax in focus.

### 3. How do I get more themes?
I can see the global config has a bunch of colour variables. Does the syntax match any other theming conventions? Or is there a place where I can get a different theme for the app?

I just wanna say this is a great app and a perfect balance between pure shortcuts and cofig files and essential ui elements that make it more user friendly. It is feindishly fast and super light. Kudos to everyone who worked on it!

**Comment by focus-editor** [comment]
> How to change default window size?

There's no way to do that currently. I've added it to the list.

> Is there an option to add language?

Right now you have to extend the editor in order to add syntax highlighting for a new language, but we're going to change that - see #105 

> How do I get more themes?

There will be predefined themes in the future, but that work is relatively low priority, because it's easy to customize the theme already.
